### ISSUE TITLE
The path doesn't contain the bundled gems anymore when we go away

### ISSUE THREAD

**Issue by janko** [tpope/vim-bundler#27]
TL;DR: Is it possible for vim-bundler to look in for the Gemfile in `:pwd`, rather than relative to the currently opened project?

I open my project in Vim. Then I `:find rails`. And then I try to `gf` under `active_support`, but I can't. When I look at Vim's `path`, I see that my gems aren't there anymore. I assume that's because vim-bundler looks for the Gemfile relative to the currently opened project (Railties, in this case). And the `railties` gem, like most others, doesn't include **their** Gemfile in the installed gem, so I can't `:find` ActiveSupport.

Could vim-bundler look for the Gemfile in `:pwd`, which is always equal to the working directory?

**Comment by nikhgupta** [comment]
I wanted exactly the same behaviour, i.e. when I navigate away to Gem directories from my Rails project, I should be able to dig deeper into the tags. Here is what I am doing (o'course this is hackish):

    augroup gem_ctags
      au!
      let $GEMPATH__LOCAL = system("gem env home")[:-2] . "/gems"
      au BufEnter $GEMPATH__LOCAL/**/*.rb setl tags+=$GEMPATH__LOCAL/*/tags
      au BufLeave $GEMPATH__LOCAL/**/*.rb setl tags-=$GEMPATH__LOCAL/*/tags
    augroup END

**Comment by tpope** [comment]
A slightly cleaner version of that might use something like `let &l:tags = getbufvar('#', 'tags')`, which steals the tags list from the previous buffer. You could do the same for `'path'` too. A really clean solution (and one that I would consider merging) would check the Bundler project of the previous buffer to see if the current file belongs to one of its gems.

**Comment by tpope** [comment]
I took a slightly closer look at this and accidentally ended up implementing a solution. Very lightly tested so let me know how it works.

**Comment by JohnMorales** [comment]
:laughing: "accidentally ended up implementing a solution" 

thanks Tim!

**Comment by janko-m** [comment]
Works for me, thank you!
